.travis.yml

```diff
@@ -6,15 +6,12 @@ env: DJ_KEY=testkey
 before_install:
 - nvm use 6
```

Comment:
After @anselmbradford 's modernizations, I don't think we need to specify node 6 anymore. the frontend build works for me with node 8.

Reply:
Yes!

Reply:
Thanks, addressed in cebb17d.